src/views/clusteroverview/SettingsTab/ExpandSection/ExpandSection.tsx

```diff
@@ -1,21 +1,28 @@
-import React, { FC, useState } from 'react';
+import React, { FC, ReactNode, useState } from 'react';
 
 import { ExpandableSection } from '@patternfly/react-core';
 
 type ExpandSectionProps = {
   className?: string;
-  toggleText: string;
+  toggleContent?: ReactNode;
+  toggleText?: string;
 };
 
-const ExpandSection: FC<ExpandSectionProps> = ({ children, className, toggleText }) => {
+const ExpandSection: FC<ExpandSectionProps> = ({
+  children,
+  className,
+  toggleContent,
+  toggleText,
+}) => {
   const [isExpanded, setIsExpanded] = useState(false);
 
   return (
     <ExpandableSection
       className={className}
       isExpanded={isExpanded}
       onToggle={setIsExpanded}
-      toggleText={toggleText}
+      toggleContent={toggleContent && toggleContent}
+      toggleText={toggleText && toggleText}
```

Comment:
can set default value to "" in the props? same for line above

Reply:
Done!